api/v1beta1/rollout_types.go

```diff
@@ -558,29 +558,29 @@ const (
 type FinalisingStepType string
 
 const (
-	// some work that should be done before pod scaling down.
-	// For BlueGreenStrategy:
-	// we rout all traffic to stable or new version based on FinaliseReason
-	// For CanaryStrategy:
-	// we remove the selector of stable service
-	FinalisingStepTypePreparing FinalisingStepType = "Preparing"
+	// Route all traffic to stable or new version based on FinaliseReason (for bluegreen)
+	FinalisingStepTypeRouteAllTraffic FinalisingStepType = "RouteAllTraffic"
```

Comment:
is FinalisingStepTypeRouteAllTraffic and FinalisingStepTypeTrafficRouting ever used now?

Reply:
they are used by the bluegreen strategy